src/haddock/modules/refinement/emref/__init__.py

```diff
@@ -108,7 +110,7 @@ def run(self, **params):
                 self.path,
                 self.recipe_str,
                 self.params,
-                ambig_f=self.params.get('ambig', None),
+                ambig=self.params.get('ambig', None),
```

Comment:
There is a bug here with `ambig` not being `ambig_fname`. I will correct that and inspect also other things. Please wait before merging.